eoc_journal/eoc_journal.py

```diff
@@ -134,9 +192,15 @@ def _get_current_user(self):
         """
         xblock_user = self.runtime.service(self, 'user').get_current_user()
         user_id = xblock_user.opt_attrs['edx-platform.user_id']
-        user = User.objects.get(pk=user_id)
+        user = User.objects.get(pk=user_id)  # pylint: disable=no-member
         return user
 
+    def _get_current_anonymous_user_id(self):
+        """
+        Returns string corresponding to the current user.
```

Comment:
nit: "Returns anonymous id (string) corresponding to the current user." would be better.